### PR TITLE
Fix ows chart for v3

### PR DESCRIPTION
**Pull request by pindge**
We'll clean up the other folder when this is migrated

### CODE REVIEW

File: stable/datacube-ows/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- http
+- web
+- application
+- python
+home: https://www.opendatacube.org/documentation
+# icon:
+sources:
+- https://github.com/opendatacube/datacube-core
+- https://github.com/opendatacube/datacube-ows
+# maintainers:
```
Comment (whatnick) [inline stable/datacube-ows/Chart.yaml:16]: @pindge please take ownership as maintainer of this Chart 👍

File: stable/datacube-ows/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+[Datacube](https://www.opendatacube.org/) Web Map Service
+
+## Example Config
+
+This example works on docker for mac, with a pre-indexed postgres database hosted on the host machine
+
+```yaml
+image:
+  registry: docker.io
+  repository: opendatacube/wms
```
Comment (whatnick) [inline stable/datacube-ows/README.md:12]: Point to new repo.

File: stable/datacube-ows/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+service:
+  type: LoadBalancer
+
+minReplicas: 1
+maxReplicas: 2
+
+wms:
+  config:
+    url: https://raw.githubusercontent.com/GeoscienceAustralia/dea-config/master/dev/services/wms/ows/wms_cfg.py
```
Comment (whatnick) [inline stable/datacube-ows/README.md:40]: Point to new mount based config.

File: stable/datacube-ows/Chart.yaml (hunk 1 above)
Comment (pindge) [reply stable/datacube-ows/Chart.yaml:16]: lol? really?

**Pull request merged**